app/qml/layers/LayerDetail.qml

```diff
@@ -202,6 +202,33 @@ Page {
               }
             }
           }
+
+          Column {
+            id: layerAttribution
+
+            Layout.fillWidth: true
+            Layout.preferredHeight: attributionText.height + attributionTitle.height + 2 * InputStyle.panelMarginV2
+
+            spacing: InputStyle.panelSpacing
+
+            Text {
+              id: attributionTitle
+
+              text: qsTr( "Attribution" )
+              font.bold: true
+              font.pixelSize: InputStyle.fontPixelSizeNormal
+            }
+
+            Text {
+              id: attributionText
```

Comment:
Shall we wrap the attribution into a gray rectangle like with the visibility and legend?